cockroach/test_client.py

```diff
@@ -70,7 +70,8 @@ def test_retry_non_txn(self):
             # Write/write conflicts.
             (Methods.Put, data_pb2.SNAPSHOT, True, 2),
             (Methods.Put, data_pb2.SERIALIZABLE, True, 2),
-            # The write/write can't-push test cases take 15 seconds each.
+            # Some of the can't-push test cases take 15 seconds each,
+            # and are currently failing when uncommented.
```

Comment:
it has to do with the RetryOptions, see the issue here. The corresponding test in Cockroach needs a rewrite as it relies on getting a WriteIntentError back when running against a running txn (but the retry happens in store, so it will not return to the client until it succeeds).